### ISSUE TITLE
Param PTAMMapLock does not update

### ISSUE THREAD

**Issue by joseparnau** [thinclab/uga_tum_ardrone#5]
Hi,

I have been toggling on and off the LockMap using "p toggleLockMap". This seems to work as the GUI plots when it has been locked ("PTAM map locked.") or unlocked ("PTAM map UNlocked").

I am also monitoring the ROS parameters through:
$ rosparam get /drone_stateestimation/PTAMMapLock
but the parameter /drone_stateestimation/PTAMMapLock never changes.

It should change, right?

**Comment by kbogert** [comment]
No, the tum_ardrone programmers never correctly implemented this.  Most parameters are internal to the package only and not exposed properly to ROS.

**Comment by joseparnau** [comment]
ok, thx!
